network/substreams.yaml

```diff
@@ -1,68 +1,97 @@
 specVersion: v0.1.0
 package:
-  name: "messari_networks_substreams"
+  name: messari_networks_substreams
   version: v0.1.0
 
 imports:
-  eth: https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg
-
-protobuf:
-  files:
-    - network.proto
-  importPaths:
-    - ./proto/v1
+  eth: "https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg"
 
 binaries:
   default:
     type: wasm/rust-v1
-    file: ../target/wasm32-unknown-unknown/release/messari_networks_substreams.wasm
+    file: "../target/wasm32-unknown-unknown/release/network.wasm"
 
 modules:
-  - name: store_cumulative_values
+  - name: unique_tracking
```

Comment:
Do we want to move away from prepending each module with `map_` or `store_`?

Reply:
I think as long as it's clear what the module is doing I think it's ok. I picked this for the module name as I thought this was the clearest thing to sum up what this module is doing, i.e., Tracking uniqueness among a few fields.

Reply:
sounds good, will keep that in mind during development